frontend/.gitignore

```diff
@@ -23,4 +23,4 @@ pnpm-debug.log*
 *.sw?
 /.env.local
 /reports/coverage/
-/env.local
+/.env.local
```

Comment:
already declared in line 24. redo, this is to make sure it isnt leaked if it isnt named properly